django/contrib/admin/options.py

```diff
@@ -949,6 +949,26 @@ def get_list_display_links(self, request, list_display):
             # Use only the first item in list_display as link
             return list(list_display)[:1]
 
+    def get_field_help_text(self, field_name):
+        """
+        Return the help text property of an item in list_display.
+        """
+        try:
+            field = self.model._meta.get_field(field_name)
+            return getattr(field, 'help_text', None)
+        except FieldDoesNotExist:
+            method = None
+            for item in self.list_display:  # Looking for callable
+                if callable(item) and item.__name__ == field_name:
+                    method = item
+                    break
+            if not method:
+                method = getattr(self, field_name, None)  # ModelAdmin method
+            if not method:
+                method = getattr(self.model, field_name, None)  # Model method
+            if method:
+                return getattr(method, 'help_text', None)
```

Comment:
I think this logic should be moved to the `django.contrib.admin.utils.help_text_for_field()` this will allow reusing it in other places (like we do with `short_description` in `label_for_field()`).

Reply:
I moved it to `django.contrib.admin.utils.help_text_for_field()`